package.json

```diff
@@ -21,6 +21,10 @@
     "e2e:smoke": "playwright test src/e2e/ --grep @smoke",
     "cron:monthly-activity": "node dist/scripts/cronjobs/monthlyActivity.js",
     "cron:breach-alerts": "node src/scripts/emailBreachAlerts.js",
+    "cron:db-delete-unverified-subscribers": "node scripts/delete-unverified-subscribers.js",
+    "cron:db-pull-breaches": "node src/scripts/syncBreaches.js",
+    "cron:remote-settings-pull-breaches": "node scripts/updatebreaches.js",
+    "cron:onerep-limits-alert": "node src/scripts/onerepStatsAlert.js",
```

Comment:
nit: should the script name match the filename, or should the filename match the script name?
ie:
```js
"cron:onerep-stats-alert": "node src/scripts/onerepStatsAlert.js",
// vs
"cron:onerep-limits-alert": "node src/scripts/onerepLimitsAlert.js",
```

Or I see one script above is in a /cronjobs/ directory, although its in a dist/ folder, so maybe that is transpiled, and these scripts shouldn't move into a /cronjobs/ directory to make their intent a bit clearer (at the sake of complicating the PR).

Reply:
Which cronjobs we run are configured elsewhere, and they're currently pointing to these script paths directly. After this PR, I'm hoping to point them to these npm scripts, and then we can update the paths to the other cronjobs with independent PRs in just this repo - both to move them into a `/cronjobs/` directory, as well as to enable TypeScript (and thus compilation).